internal/address/database.go

```diff
@@ -0,0 +1,44 @@
+package address
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+type ZipDatabase map[string]Address
+
+type Address struct {
+	ZipCode         string   `json:"cep"`
+	State           string   `json:"state"`
+	InitialDistrict District `json:"initialDistrict"`
+	FinalDistrict   District `json:"finalDistrict"`
+	AddressLine     string   `json:"addressLine"`
+	City            string   `json:"city"`
+	Deleted         string   `json:"deleted"`
+}
+
+type District struct {
+	Key  int    `json:"key"`
+	Name string `json:"name"`
+}
+
+func FromJson(jsonData io.Reader) (database ZipDatabase, err0 error) {
```

Comment:
```suggestion
func FromJson(jsonData io.Reader)  ZipDatabase,  error) {
```
You don't normally use named return values if you are not going to use them.